.copr/Makefile

```diff
@@ -0,0 +1,64 @@
+# Copyright David Cantrell <dcantrell@redhat.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Set the top level source directory
+topdir := $(shell realpath $(dir $(lastword $(MAKEFILE_LIST)))/..)
+
+# Install packages before anything else
+_install := $(shell dnf install -y git)
+_safedir := $(shell git config --global --add safe.directory $(topdir))
+
+# Pass BUILDTYPE=release to generate a release SRPM
+BUILDTYPE ?= copr
+
+# Spec file and template
+SPEC_TEMPLATE = $(shell ls -1 $(topdir)/*.spec)
+SPEC = $(topdir)/$(shell basename $(SPEC_TEMPLATE))
+
+# Replace placeholders in the spec file template
+RPMDATE = $(shell date +'%a %b %d %Y')
+#RPMAUTHOR = $(shell git log | grep ^Author: | head -n 1 | cut -d ' ' -f 2,3,4)
+RPMAUTHOR = David Cantrell <dcantrell@redhat.com>
+
+# Various things we need to generate a tarball
+PKG = $(shell rpmspec -P "$(SPEC_TEMPLATE)" | grep ^Name: | awk '{ print $$2; }')
+VER = $(shell rpmspec -P "$(SPEC_TEMPLATE)" | grep ^Version: | awk '{ print $$2; }')
+
+ifeq ($(BUILDTYPE),copr)
+GITDATE          = $(shell date +'%Y%m%d%H%M')
+GITHASH          = $(shell git rev-parse --short HEAD)
+TARBALL_BASENAME = $(PKG)-$(VER)-$(GITDATE)git$(GITHASH)
+TAG              = HEAD
+else
+0TAG             = $(shell git tag -l | sort -V | tail -n 1)
```

Comment:
The returned git tag is wrong (maint-el7-0.6.4-1-build-sync). Fortunately 0TAG is not used for anything.

Reply:
@dcantrell probably copied that snippet as boilerplate from another project. And `0TAG` is probably a typo of `TAG`... see https://github.com/rpminspect/rpminspect/blob/main/.copr/Makefile. I've changed it to `TAG = $(shell git describe --tags --abbrev=0)` which gets the most recent tag in history rather than the tag that sorts the highest, and therefore works better with the dnf repository.

Reply:
Yes, 0TAG was a typo.  Sorry.